lib/gui/models/store.js

```diff
@@ -91,7 +105,8 @@ const storeReducer = (state, action) => {
 
       const newState = state.set('availableDrives', Immutable.fromJS(action.data));
```

Comment:
I've never written any Redux code before, but should the first parameter to these `.set` calls also be constants, rather than just strings, to cut down on the possibility of any typos?